.gitignore

```diff
@@ -69,3 +69,14 @@ go.work.sum
 
 # Development containers (https://containers.dev/)
 .devcontainer
+
+# ironic username and password files
+test/e2e/data/*/overlays/*/ironic-username
```

Comment:
Put these in a gitignore closer to the ignored directory. Here that'd be in `test/e2e/data`.